build-local.sh

```diff
@@ -15,8 +15,6 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-[ -z "$SUDO_COMMAND" ] && exec sudo $0
-
 source ./vars.sh
 
 if ! type buildah; then
```

Comment:
pls run from registry.fedoraproject.org/fedora:29 so that we dont pull random bits off the internet